test/common/endian.h

```diff
@@ -45,4 +45,23 @@ class EndianTestSuite : public CxxTest::TestSuite
 		uint32 value = READ_LE_UINT16(data);
 		TS_ASSERT_EQUALS(value, 0x3412UL);
 	}
+
```

Comment:
I know the code is simpler for float32 but it may not hurt to have some tests for them too.

Reply:
OK, done.